src/main/resources/org/jenkinsci/plugins/workflow/steps/ToolStep/help-name.html

```diff
@@ -0,0 +1,3 @@
+<div>
+ The name of the tool. The tool name must be pre-configured in Jenkins under <em>Manage Jenkins</em> → <em>Global Tool Configuration</em>.
```

Comment:
In my opinion, it's better to use HTML code here rather than this character.

```suggestion
 The name of the tool. The tool name must be pre-configured in Jenkins under <em>Manage Jenkins</em> &rarr; <em>Global Tool Configuration</em>.
```